.github/workflows/ci_tests.yml

```diff
@@ -46,59 +49,92 @@ jobs:
           - description: "ASan"
             args: "-DCMAKE_CXX_FLAGS='-fsanitize=address -fsanitize=undefined'"
         include:
-          - platform: ubuntu-latest
-            compiler:
+          - platform:
+              description: "ubuntu gcc"
               cpp: g++
               c: gcc
+              os: ubuntu-latest
             cpp_version: 17
             cmake_args:
               description: "Werror"
               args: "-DCMAKE_CXX_FLAGS='-Werror=all -Werror=extra'"
-          - platform: ubuntu-latest
-            compiler:
+          - platform:
+              description: "ubuntu gcc"
               cpp: g++
               c: gcc
+              os: ubuntu-latest
             cpp_version: 17
             cmake_args:
               description: "Dynamic"
               args: "-DBUILD_SHARED_LIBS=on"
+            # MSVC has limited support for sanitizers, so we use include here for MSVC as an exception.
+            # This should be simplified after beman-project/exemplar#76 is merged.
```

Comment:
```suggestion
```

It isn't clear what is meant by "use include here". Also, it is unclear if #76 in its current form will be merged.

Reply:
yeah sorry I should check my comment super well.